### PR TITLE
escape character on search

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

escape character to avoid error on package search

Fixes # (THEEDGE-2871)

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by omaciel — DISMISSED** [review]
/lgtm

File: pkg/clients/imagebuilder/client_test.go [hunk 1]
```diff
@@ -84,6 +84,18 @@ var _ = Describe("Image Builder Client Test", func() {
 		Expect(err).To(BeNil())
 		Expect(res.Meta.Count).To(Equal(0))
 	})
+	It("test validation of special character package name", func() {
+		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "application/json")
+			w.WriteHeader(http.StatusOK)
+			fmt.Fprintln(w, `{"meta":{"count":1}}`)
```
Comment (ldjebran) [inline pkg/clients/imagebuilder/client_test.go:91]: but we are not asserting that we received the right package name?

**Review by omaciel — APPROVED** [review]
/lgtm

**Review by ldjebran — APPROVED** [review]
/LGTM

**Pull request merged**